run.go

```diff
@@ -53,6 +55,17 @@ func IsInteractiveCall(rootCmd *cobra.Command, args []string) bool {
 }
 
 func RunCmdInteractively(ctx context.Context, rootCmd *cobra.Command, cnfg *hazelcast.Config) {
+	cmdHistoryPath := filepath.Join(file.HZCHomePath(), ".hzc_history")
+	exists, err := file.FileExists(cmdHistoryPath)
+	if err != nil {
+		rootCmd.Printf("Error: Can not read command history file on %s, may be missing permissions:\n%s\n", cmdHistoryPath, err)
```

Comment:
I think it's OK to be silent about history file not read. Or when we have log levels enabled, we could display them in the debug level.

Reply:
Replaced with a todo https://github.com/hazelcast/hazelcast-commandline-client/pull/66/commits/fe6bf1e1b2ca7dd456aff72357e77644aca95af7